linker/Linker.Steps/MarkStep.cs

```diff
@@ -1540,6 +1540,12 @@ protected virtual void MarkMethodBody (MethodBody body)
 
 			foreach (Instruction instruction in body.Instructions)
 				MarkInstruction (instruction);
+
+			MarkSomethingUsedViaReflection ("GetConstructor", MarkConstructorsUsedViaReflection, body.Instructions);
+			MarkSomethingUsedViaReflection ("GetMethod", MarkMethodsUsedViaReflection, body.Instructions);
+			MarkSomethingUsedViaReflection ("GetProperty", MarkPropertyUsedViaReflection, body.Instructions);
+			MarkSomethingUsedViaReflection ("GetField", MarkFieldUsedViaReflection, body.Instructions);
+			MarkSomethingUsedViaReflection ("GetEvent", MarkEventUsedViaReflection, body.Instructions);
 		}
```

Comment:
It'd be also nice to extend this to support extension methods on RuntimeReflectionExtensions (e.g. GetRuntimeMethod)

Reply:
I agree, although I think that is outside the scope of this PR.